core/jobs/transforms/validation/blog_validation.py

```diff
@@ -80,14 +90,26 @@ def _get_domain_object_validation_type(self, unused_item):
         return base_validation.ValidationModes.STRICT
 
 
+# TODO(#15613): Due to incomplete typing of apache_beam library and absences
+# of stubs in Typeshed, MyPy assuming DoFn class is of type Any. Thus to avoid
+# MyPy's error (Class cannot subclass 'DoFn' (has type 'Any')) , we added an
+# ignore here.
 @validation_decorators.AuditsExisting(
     blog_models.BlogPostModel,
     blog_models.BlogPostSummaryModel)
-class ValidateModelPublishTimestamps(beam.DoFn):
+class ValidateModelPublishTimestamps(beam.DoFn):  # type: ignore[misc]
     """DoFn to check whether created_on and last_updated timestamps are valid.
     """
 
-    def process(self, input_model):
+    def process(
+        self, input_model: blog_models.BlogPostModel
+    ) -> Iterator[
```

Comment:
@sahiljoster32 input_model can also be of the type BlogPostSummaryModel not just BlogPostModel.  (line numbers 97-99)

Reply:
As per the offline discussion, Rijuta is going to update the type in his PR.
Thanks!!